components/dialog/dialog-mixin.js

```diff
@@ -19,7 +19,7 @@ window.D2L.DialogMixin = window.D2L.DialogMixin || {};
 // https://bugs.webkit.org/show_bug.cgi?id=233320
 // starting in Chrome 102, all elements inside <dialog>s that are inside shadow roots have null offsetParent
 // https://bugs.chromium.org/p/chromium/issues/detail?id=1331803
-window.D2L.DialogMixin.hasNative = false && (window.HTMLDialogElement !== undefined)
+window.D2L.DialogMixin.hasNative = false && (window.HTMLDialogElement !== undefined) //eslint-disable-line no-constant-binary-expression
```

Comment:
Maybe we should just set this to `false`? We were trying to remember the previous logic, but it might be best in the comment. Thoughts?

Reply:
I was thinking along those lines too. I even think commenting the logic is not ideal, we can just remove the logic and use git history when we need it.